### PR TITLE
Update Vagrant box to use Stretch

### PR DESCRIPTION
**Pull request by jacksonj04**
* Part of #2596

### CODE REVIEW

**Comment by sagepe** [comment]
So I think this is pretty much ready now. Tests pass locally, although there seems to be an issue queueing at Travis atm.

Could possibly tidy up some of the commits a bit with some judicious squashing, depends on what people think.

File: bin/vagrant-install.bash [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    echo "Unknown country, ${COUNTRY_APP}"
+    LIVE_URL=
+  ;;
+esac
+
+if [ -n "$LIVE_URL" ]; then
+  echo "Downloading and loading database dump for ${COUNTRY_APP}..."
+  TMP_DATA=$(mktemp /var/tmp/data.XXXXX)
+  curl -o ${TMP_DATA} ${LIVE_URL}/media_root/dumps/pg-dump_data.sql.gz
+  gunzip -c ${TMP_DATA} | psql pombola
```
Comment (chrismytton) [inline bin/vagrant-install.bash]: Might be good to use the `$DB_NAME` variable here as well, for consistency.

File: BACKGROUND.md [hunk 2]
```diff
@@ -59,31 +59,41 @@ Africa).
...
-The Vagrant box should set up an environment in which this
-works.
+The provisioning script used to download the public database
+dumps which are published, for example,
+[here](https://www.pa.org.za/help/api) for each country and set
+up a database for each.
 
-## Vagrant
+This is no longer the default and can now be done by running, e.g.,
+`bin/vagrant-load-data kenya` from within the vagrant box.
```
Comment (chrismytton) [inline BACKGROUND.md]: There doesn't seem to be a `bin/vagrant-load-data` script on this branch, and from what I can see the provisioning does still download the public database dumps. So I think either these docs, or the provisioning script need to be updated 🙂

File: docs/INSTALL.md [hunk 3]
```diff
@@ -38,10 +31,11 @@ Databases
 ---------
 A Postgis enabled Postgres database is required
 
-Create the database - assuming here that you're calling it 'pombola'
+Create the database and enable the Postgis extentions- assuming here that you're calling it 'pombola'
```
Comment (chrismytton) [inline docs/INSTALL.md]: Typo: extentions ➡️ extensions

File: docs/OVERVIEW.md [hunk 4]
```diff
@@ -17,9 +17,8 @@ There is also a sample Apache2 config in 'conf/httpd.conf-example' which might
 be helpful.
 
 When installing the system note that many additional files and directories will
-be created alongside the top level git repo checkout. These include the
-'media_root' for uploaded images, and the 'pombola_xapian' for the search
-index.
+be created in the `data/` diorectory in the top level git repo checkout. These
```
Comment (chrismytton) [inline docs/OVERVIEW.md]: Typo: diorectory ➡️ directory

File: bin/vagrant-install.bash (hunk 1 above)
Comment (sagepe) [reply bin/vagrant-install.bash]: Done.

File: BACKGROUND.md (hunk 2 above)
Comment (sagepe) [reply BACKGROUND.md]: Oops, yes - I removed this. Updated to reflect.

File: docs/INSTALL.md (hunk 3 above)
Comment (sagepe) [reply docs/INSTALL.md]: Fixed.

File: docs/OVERVIEW.md (hunk 4 above)
Comment (sagepe) [reply docs/OVERVIEW.md]: Fixed.

**Comment by chrismytton** [comment]
This is working really well, thanks for your work on this @sagepe! 🙏

**Pull request merged**